pulumi/grapl/__main__.py

```diff
@@ -134,9 +134,22 @@ def main() -> None:
     nomad_provider: pulumi.ProviderResource | None = None
     consul_provider: pulumi.ProviderResource | None = None
 
+    # Use the nomad ALB address or default to the local host address.
+    # Apparently nomad templates use a different templating syntax than anywhere else
+    nomad_endpoint: pulumi.Output[
+        str
+    ] | str = '{{ env "attr.unique.network.ip-address" }}:4646'
+
     if not config.LOCAL_GRAPL:
         upstream_stacks = UpstreamStacks()
         nomad_provider = get_nomad_provider_address(upstream_stacks.nomad_server)
+
+        nomad_endpoint = nomad_provider.address
```

Comment:
Do we need this intermediate variable? Could we just do the processing directly on `nomad_provider.address`?

Reply:
No we don't, fixing now.